changelogs/unreleased/prbuf.md

```diff
@@ -0,0 +1,4 @@
+## feature/lua
+
+* Introduced new buffer - partitioned ring buffer, which is available in Lua
+  via require('buffer').prbuf_create() and require('buffer').prbuf_open().
```

Comment:
I'm not sure we should make it public. Moreover, I don't even see how we could use it internally. Why do you need it?

Reply:
Okay. It'd be nice to mention in the commit message. Still, I don't think we need to document it or mention it in the changelog, at least now, when nobody asked for it. We might want to change the prbuf API in the future. Making the API public would render such changes complicated.

Reply:
Flight recorder is an enterprise feature, so that I've skipped mentioning it in the commit message.